package.json

```diff
@@ -58,7 +57,8 @@
     "typescript": "^3.8.3",
     "worker-farm": "^1.5.0",
     "wtfnode": "^0.8.0",
-    "yargs": "^14.2.0"
+    "yargs": "^14.2.0",
+    "util.promisify": "^1.0.1"
```

Comment:
nit: keep the dependencies in alphabetical order

Reply:
fixed